config/locales/en.yml

```
@@ -2032,6 +2032,114 @@ en:
       users:
         older: Older Users
         newer: Newer Users
+    language_selector:
+      af: Afrikaans
+      aln: Albanian (Gheg)
+      ar: Arabic
+      arz: Arabic (Egyptian)	
+      ast: Asturian
+      az: Azerbaijani
+      ba: Bashkir
+      be-Tarask: Belarusian (Taraškievica)
+      be: Belarusian
+      bg: Bulgarian
+      bn: Bengali
+      br: Breton
+      bs: Bosnian
+      ca: Catalan, Valencian
+      ce: Chechen
+      cs: Czech
+      cy: Welsh
+      da: Danish
+      de: German
+      diq: Dimli (individual language)
+      dsb: Lower Sorbian
+      el: Greek
+      en-GB: English (United Kingdom)
+      en: English
+      eo: Esperanto
+      es: Spanish, Castilian
+      et: Estonian
+      eu: Basque
+      fa: Persian
+      fi: Finnish
+      fit: Finnish (Tornedalen)
+      fr: French
+      fur: Friulian
+      fy: Western Frisian
+      ga: Irish
+      gcf: Guadeloupean Creole French
+      gd: Gaelic, Scottish Gaelic
+      gl: Galician
+      gsw: Alemannic (specifically Swiss German, Alsatian)
+      gu: Gujarati
+      he: Hebrew
+      hi: Hindi
+      hr: Croatian
+      hsb: Sorbian, Upper
+      hu: Hungarian
+      ia: Interlingua
+      id: Indonesian
+      is: Icelandic
+      it: Italian
+      ja: Japanese
+      ka: Georgian
+      kab: Kabyle
+      kk-cyrl: Kazakh (Cyrillic)
```

If we are to use this approach (ask translators to do n x n language name translations), the keys here have to be whatever `I18n::Locale::Tag::Rfc4646` normalizes them to, see https://github.com/openstreetmap/openstreetmap-website/pull/5201#discussion_r2084730324.

This is the change for Kazakh:
```suggestion
      kk-Cyrl: Kazakh (Cyrillic)
```